atom/browser/api/atom_api_web_contents.cc

```diff
@@ -1760,9 +1760,7 @@ bool WebContents::IsLoading() const {
 bool WebContents::IsLoadingMainFrame() const {
   // Comparing site instances works because Electron always creates a new site
   // instance when navigating, regardless of origin. See AtomBrowserClient.
-  return (web_contents()->GetLastCommittedURL().is_empty() ||
-          web_contents()->GetSiteInstance() !=
-          web_contents()->GetPendingSiteInstance()) && IsLoading();
+  return web_contents()->IsLoadingToDifferentDocument();
```

Comment:
@bridiver I was about to ask what tests to perform to check if the new code is equivalent, but I see it isn't used at all in `browser-laptop`, can we remove it?